### PR TITLE
Add job types for checkJobHealth and defaultJobs

### PR DESCRIPTION
**Pull request by KhushbuFuletra**
Add default job types to include QueuedJob::QUEUED, SqsQueuedJobExtension::TYPE_SCHEDULED
This change is for SS3 version (~2.2.0) and NOT to be merged into master which belongs to SS4 version.

### CODE REVIEW

File: code/queuedjobs/SqsScheduleRunnerJob.php [hunk 1]
```diff
@@ -25,8 +25,10 @@ class SqsScheduleRunnerJob implements SqsIntervalTask {
 
 
     public function processScheduledJobs() {
-        $this->queuedJobService->checkJobHealth();
-        $this->queuedJobService->checkdefaultJobs();
+        $queue = array(QueuedJob::QUEUED, SqsQueuedJobExtension::TYPE_SCHEDULED);
```
Comment (nyeholt) [inline code/queuedjobs/SqsScheduleRunnerJob.php]: Looking at some of the issues that have been causing blocked queues, this should also check for QueuedJob::LARGE

**Review by nyeholt — COMMENTED** [review]
One comment about the job types, but Github is also mentioning a conflict somewhere?

File: code/queuedjobs/SqsScheduleRunnerJob.php (hunk 1 above)
Comment (KhushbuFuletra) [reply code/queuedjobs/SqsScheduleRunnerJob.php]: @nyeholt Added QueudJob::LARGE.

**Comment by KhushbuFuletra** [comment]
> One comment about the job types, but Github is also mentioning a conflict somewhere?
You are seeing the conflict because this is an SS3 change, it can't be merged back into master, it has to go into tagged version 2.2.3. You might need to cherry-pick and merge this. Github doesn't allow merging into old tags. 
OR in SS3 projects we can use my forked repository

**Pull request merged**